package.nls.json

```diff
@@ -46,6 +46,9 @@
     "config.misc.color3Picker": "Enable Color Picker and Preview for Color3 functions.",
     "config.misc.serviceAutoImport": "Suggests Roblox Services that can be auto imported.",
     "config.misc.goToScriptLink": "If enabled, shows a link to the source script when requiring a module.",
+    "config.misc.importScriptChildren": "Suggest require paths that index into scripts.",
+    "config.misc.importIgnore": "Globs to never suggest when suggesting require paths. For example: `**/_Index/**` to ignore wally's index folder. Case sensitive. Acts on the absolute path of files.",
+    "config.misc.importPathType": "Whether to show relative or absolute require paths.",
```

Comment:
Add descriptions to enums. For example: 
```json
"config.misc.importPathType.Only Relative Paths": "Only import paths relative to the current script."
```

Reply:
Ah, I didn't realize this was possible! Will do.